File: 6-kyu/multiples-of-3-or-5/multiples-of-3-or-5.java
public class Solution {
​
  public int solution(int number) {
    int sum = 0;
    number --;
    if(number <= 0) return 0;
    while(number > 0){
      if(number % 3 == 0 || number % 5 == 0) sum += number;
      number --;
    }
    return sum;
  }
}

File: 6-kyu/multiples-of-3-or-5/multiples-of-3-or-5.py
def solution(number):
    number -= 1
    sum = 0
    if number <= 0:
        return 0
​
    while number > 0:
        sum += number if (number % 3 == 0 or number % 5 == 0) else 0
        number -= 1
    return sum